mkdocs.yml

```diff
@@ -45,74 +45,75 @@ markdown_extensions:
 nav:
   - index.md
   - Start Here:
-    - getting-started.md
-    - connect.md
-    - try-first.md
-    - finding-commands.md
-    - eval-tips.md
+      - getting-started.md
+      - connect.md
+      - try-first.md
+      - finding-commands.md
+      - eval-tips.md
   - why-calva.md
   - Features:
-    - commands-top10.md
-    - evaluation.md
-    - formatting.md
-    - syntax-highlighting.md
-    - paredit.md
-    - REPL:
-      - repl-window.md
-      - output.md
-      - inspector.md
-      - fiddle-files.md
-      - connect-sequences.md
-      - custom-commands.md
-      - flares.md
-    - refactoring.md
-    - notebooks.md
-    - clojuredocs.md
-    - debugger.md
-    - pprint.md
-    - test-runner.md
-    - rich-comments.md
-    - hiccup.md
+      - commands-top10.md
+      - evaluation.md
+      - formatting.md
+      - syntax-highlighting.md
+      - paredit.md
+      - REPL:
+          - repl-window.md
+          - output-webview.md
```

Comment:
This was added. Just noting that since there are whitespace changes which might make that not obvious.